client/src/components/common/Card/cardContent.js

```diff
@@ -1,29 +1,43 @@
+/* eslint-disable jsx-a11y/no-static-element-interactions */
 import React from 'react';
-import propTypes from 'prop-types';
+import PropTypes from 'prop-types';
 import { Avatar } from 'antd';
 
 import './style.css';
 
-const CardContent = ({ avatar, approach, type }) => (
-  <div className="card__content">
-    <Avatar
-      shape="circle"
-      src={avatar}
-      size={100}
-      icon="user"
-      className="card__photo"
-    />
-    <div>
-      <h2 className="card__approach">{approach} </h2>
-      <h3 className="card__type">{type}</h3>
-    </div>
-  </div>
-);
+export default class CardContent extends React.Component {
+  handleClick = () => {
+    const { id } = this.props;
+    // eslint-disable-next-line react/destructuring-assignment
+    this.props.props.history.push(`/profile/${id}`);
+  };
+
+  render() {
+    const { image, approach, type, city } = this.props;
+    return (
+      <div className="card__content" onClick={this.handleClick}>
```

Comment:
To resolve the interaction eslint error above
```suggestion
      <div role="button" tabIndex={0} className="card__content" onClick={this.handleClick}>
```